ansible/ansible-playbook.yml

```diff
@@ -0,0 +1,70 @@
+- name: Install To-Do App on new web server
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: SECRET_KEY
+      prompt: SECRET_KEY
+
+    - name: TRELLO_API_KEY
+      prompt: TRELLO_API_KEY
+
+    - name: TRELLO_API_TOKEN
+      prompt: TRELLO_API_TOKEN
+
+  tasks:
+    - name: Install Git
+      become: yes
+      ansible.builtin.yum:
+        name: git
+        state: present
+
+    - name: Install Python3
+      become: yes
+      ansible.builtin.yum:
+        name: python3
+        state: latest
```

Comment:
Minor, but note that we can combine these _if_ we are happy to have the same "state" for each:
```yml
    - name: Install Git & Python
      become: yes
      ansible.builtin.yum:
        name: 
          - git
          - python3
        state: present
```